templates/chisel/nix/gcd/vcs.nix

```diff
@@ -12,6 +12,8 @@
 
 let
   binName = "gcd-vcs-simulator";
+  coverageName = "coverage.vdb";
+  enableCover = true;
```

Comment:
Perhaps we could make `enableCover` an input of the derivation (with default value)?